File: URLShortener.Back/URLShortener.Models/ShortURLInfoModel.cs
namespace URLShortener.Models
{
    public class ShortURLInfoModel
    {
        public int Id { get; set; } 
        public string Url { get; set; } 
        public string Origin { get; set; } 
        public string CreatedBy { get; set; } 
        public string CreatedByUserId { get; set; } 
        public DateTime CreationDate { get; set; } 
    }
}

File: URLShortener.Front/src/app/models/ShortURLInfoModel.ts
export class ShortURLInfoModel{
    constructor(id: number, url:string,origin:string,createdBy:string,creationDate:Date){
        this.id = id;
        this.url = url;
        this.origin = origin;
        this.createdBy = createdBy;
        this.creationDate = creationDate;
    }
    public id: number = 0;
    public url: string = '';
    public origin: string = '';
    public createdBy: string = '';
    public creationDate: Date;
}